sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -58,7 +58,7 @@ your resource and a `credential` that allows you access:
 ```python
 from azure.ai.textanalytics import TextAnalyticsClient
 
```

Comment:
Also, at the top of the readme we still mention the six main functions to include `Named Entity Recognition`. Do we want to continue calling that or have it updated to Categorized?

Reply:
Good point, I think I'll change it to Categorized to reflect the SDK terminology over the service terminology

Reply:
hmm idk I'm kind of torn here. 
We should have the ML literature in our Readmes and I think in the beginning of the Readme, where we introduce what the service does, the Named Entity Recognition (NER) makes sense.

In the rest of the Readme, when we talk about how the client works, then we can use whatever name we prefer.

Reply:
See what you're saying. I agree that we should introduce the service with their terminology, I'll make that change.